packages/edit-post/src/components/sidebar/plugin-pre-publish-panel/test/index.js

```diff
@@ -11,21 +15,19 @@ import PluginPrePublishPanel from '../';
 
 describe( 'PluginPrePublishPanel', () => {
 	test( 'renders fill properly', () => {
-		const div = document.createElement( 'div' );
-		render(
+		const { container } = render(
 			<SlotFillProvider>
 				<PluginPrePublishPanel
 					className="my-plugin-pre-publish-panel"
 					title="My panel title"
-					initialOpen={ true }
+					initialOpen
 				>
 					My panel content
 				</PluginPrePublishPanel>
 				<PluginPrePublishPanel.Slot />
-			</SlotFillProvider>,
-			div
+			</SlotFillProvider>
 		);
 
-		expect( div.innerHTML ).toMatchSnapshot();
+		expect( container.innerHTML ).toMatchSnapshot();
```

Comment:
Mind going with `container` instead? We'll need to update the snapshot, but at least we'll get rid of a [`no-node-access`](https://github.com/testing-library/eslint-plugin-testing-library/blob/main/docs/rules/no-node-access.md) violation.

```suggestion
		expect( container ).toMatchSnapshot();
```

Reply:
As the purpose of the test is to check whether the `PluginPrePublishPanel` was rendered at all, by filling in a slot, I modified the assertion to check for the panel title element. Checking the entire markup with a snapshot is redundant, so I removed the snapshot completely.